tmcprototype/CspMasterLeafNode/CspMasterLeafNode/CspMasterLeafNode.py

```diff
@@ -326,6 +326,7 @@ def On(self, argin):
         :return: None
         """
         self._csp_proxy.command_inout_asynch(CONST.CMD_ON, argin, self.commandCallback)
+        # self._csp_proxy.command_inout_asynch(CONST.CMD_ON, self.commandCallback)
```

Comment:
Can we remove this print statement?

Reply:
Done.

Reply:
Commit ID: [4fe0b933c9bfe4e5c66c31e0dfae0261bf485831]